composer.json

```diff
@@ -11,14 +11,21 @@
     }
   ],
   "scripts": {
-    "test": "vendor/bin/phpunit"
+    "test": "vendor/bin/phpunit",
+    "fix-cs": "vendor/bin/php-cs-fixer fix --config=.cs.php",
+    "check-cs": "vendor/bin/php-cs-fixer fix --dry-run --format=txt --verbose --diff --diff-format=udiff --config=.cs.php",
+    "check-all": [
+      "@check-cs",
+      "@test"
```

Comment:
Define the composer scripts. They're as follows:

- `fix-cs` script is to check coding style and fix.
- `check-cs` script is to check coding style without fix.
- `check-all` scripts to do `check-cs` and `test` scripts.